app/experimenter/nimbus-ui/package.json

```diff
@@ -47,9 +47,11 @@
     "react-helmet": "^6.1.0",
     "react-hook-form": "^6.12.2",
     "react-scripts": "3.4.0",
+    "react-scrollspy": "^3.4.3",
     "react-select": "^3.1.1",
     "react-select-event": "^5.1.0",
     "react-tooltip": "^4.2.10",
+    "storybook": "^6.1.14",
```

Comment:
I tend to install storybook locally to test. This might be a different testing workflow than others use. @LZoog how do you typically run storybook locally without having this dependency?

Reply:
Good catch @pdehaan! @emtwo with the `yarn storybook` command in `nimbus-ui`... the package is `@storybook/react` and not `storybook`, you should be able to remove this dependency.